docker/centos-local-build/build.sh

```diff
@@ -70,4 +71,5 @@ rm -rf build
 rm -rf dist
 rm -f *.spec
 cd ..
-tar -czf antares-xpansion-centos.tar.gz -C $install_dir . --exclude='examples'
\ No newline at end of file
+tar -czf antares-xpansion-centos.tar.gz -C $install_dir . --exclude='examples'
+chmod 777 antares-xpansion-centos.tar.gz
```

Comment:
also remove all other files?

Reply:
which "all other files" ?

Reply:
on my PC the build folder ends up like this : 
![image](https://github.com/user-attachments/assets/64c614ca-bbc0-4645-ac6b-e60b4256bde8)

if I understand well, only the archive is needed. you can delete the 4 dirs ?